[FEATURE] Corrections de traductions ES (PIX-13523)

pix-service-prod opened the pull request:
Exported the most recent locale files from Phrase.
Corrections apportées suite aux retour de la Diputacio de Barcelona : https://1024pix.slack.com/archives/C03GHPT08TV/p1720704445683389

pix-service-prod committed:
Updated translations from Phrase

phrase: update mon-pix/translations/nl.json to latest state
phrase: update mon-pix/translations/es.json to latest state

pix-bot-github commented:
Une fois les applications déployées, elles seront accessibles via les liens suivants :
  * [App (.fr)](https://app-pr9590.review.pix.fr)
  * [App (.org)](https://app-pr9590.review.pix.org)
  * [Orga (.fr)](https://orga-pr9590.review.pix.fr)
  * [Orga (.org)](https://orga-pr9590.review.pix.org)
  * [Certif (.fr)](https://certif-pr9590.review.pix.fr)
  * [Certif (.org)](https://certif-pr9590.review.pix.org)
  * [Junior](https://junior-pr9590.review.pix.fr)
  * [Admin](https://admin-pr9590.review.pix.fr)
  * [API](https://api-pr9590.review.pix.fr/api/)
  * [Audit Logger](https://pix-audit-logger-review-pr9590.osc-fr1.scalingo.io/api/)

Les variables d'environnement seront accessibles via les liens suivants :
  * [scalingo front](https://dashboard.scalingo.com/apps/osc-fr1/pix-front-review-pr9590/environment)
  * [scalingo api](https://dashboard.scalingo.com/apps/osc-fr1/pix-api-review-pr9590/environment)
  * [scalingo audit-logger](https://dashboard.scalingo.com/apps/osc-fr1/pix-audit-logger-review-pr9590/environment)

pix-service-prod committed:
Updated translations from Phrase

phrase: update mon-pix/translations/nl.json to latest state
phrase: update mon-pix/translations/es.json to latest state

pix-service-prod committed:
Updated translations from Phrase

phrase: update mon-pix/translations/nl.json to latest state
phrase: update mon-pix/translations/es.json to latest state

jujumathieu reviewed (approved):
Ok, toutes les clés ont bien été modifiées
